src/runner.ts

```diff
@@ -156,6 +156,7 @@ async function runWorker(options: Options, logger: Logger): Promise<Status> {
     return options.force || errorCount === 0 ? Status.Ok : Status.LintError;
 }
 
+// tslint:disable-next-line:no-async-without-await
```

Comment:
ummm, how about just fixing the code?? remove the `async` keyword since it already declares it returns a Promise. same below.

Reply:
Thanks for the review. I can't remove the `tslint:disable` because it will collide with the `promise-function-async` rule. Other than that I updated everything according to your review, seems tests are passing now.